keyboards/enviousdesign/60f/info.json

```
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "EnvKB 60F V1", 
+    "url": "http://envious.design", 
+    "maintainer": "Envious.Design", 
```

```suggestion
    "manufacturer": "Envious.Design", 
```
or 
```suggestion
    "manufacturer": "Envious.Design", 
    "maintainer": "Envious.Design", 
```

Either way, it needs the manufacturer listed